src/ligoOp.ml

```diff
@@ -81,8 +89,13 @@ module Tezos = struct
 
   let get_contract_opt address = Some (contract_of_address address)
 
-  let create_contract code delegate tez store =
-    let contract = CreateContract (code, delegate, tez, store) in
+  let burrow_create_contract code delegate tez store =
+    let contract = CreateBurrowContract (code, delegate, tez, store) in
+    let address = get_next_address () in
```

Comment:
> get_next_address()

Ah, I had always wondered how we get the new addresses prior to applying the operation :sweat_smile:

Reply:
Hehe, I know, it's not all that magical or pretty when you look at it from the inside :sweat_smile: The signature matches though the LIGO one; I have no idea how tezos implements this part internally :shrug: